### PR TITLE
fix: translations - email verify and conferencing app 

### PR DESCRIPTION
**Pull request by sean-brydon**
CleanShot 2023-06-12 at 11 16 08@2x
CleanShot 2023-06-12 at 11 15 51@2x

### CODE REVIEW

**Review by alishaz-polymath — APPROVED** [review]
🚀

File: packages/emails/src/templates/VerifyAccountEmail.tsx [hunk 1]
```diff
@@ -44,13 +44,13 @@ export const VerifyAccountEmail = (
...
           <>
-            {props.language("happy_scheduling")} <br />
+            {props.language("happy_scheduling")}, <br />
             <a
               href={`mailto:${SUPPORT_MAIL_ADDRESS}`}
               style={{ color: "#3E3E3E" }}
               target="_blank"
               rel="noreferrer">
-              <>{props.language("the_calcom_team", { companyName: COMPANY_NAME })}</>
+              <>{props.language("the_calcom_team", { companyName: SENDER_NAME })}</>
```
Comment (hariombalhara) [inline packages/emails/src/templates/VerifyAccountEmail.tsx:53]: Can this be a breaking change?(for self hosters). I mean what if this env variable isn't set?

Comment (sean-brydon) [reply packages/emails/src/templates/VerifyAccountEmail.tsx:53]: Sure i will push these changes ... very good eyes @hariombalhara

**Review by pumfleet — APPROVED** [review]
Amazing, thanks!

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**